user_deprovision/user_deprovision.py

```diff
@@ -480,9 +487,9 @@ def main(args):
     )
     parser.add_argument(
         '--user-email', '-u',
```

Comment:
Can we rename this flag to be more descriptive of the fact that we are using CSV (e.g. --users-from-file)